homeassistant/components/velux/manifest.json

```diff
@@ -2,6 +2,14 @@
   "domain": "velux",
   "name": "Velux",
   "documentation": "https://www.home-assistant.io/integrations/velux",
-  "requirements": ["pyvlx==0.2.16"],
-  "codeowners": ["@Julius2342"]
-}
+  "codeowners": [
+    "@Julius2342"
+  ],
+  "requirements": [
+    "pyvlx==0.2.18"
+  ],
+  "zeroconf": [
+    "_http._tcp.local."
```

Comment:
Update this to include a name/Mac filter: https://developers.home-assistant.io/docs/creating_integration_manifest#zeroconf